sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/WriteBundlesToFiles.java

```diff
@@ -87,6 +88,28 @@ public Result(String filename, Long fileByteSize, DestinationT destination) {
       this.fileByteSize = fileByteSize;
       this.destination = destination;
     }
+
+    @Override
+    public String toString() {
+      return String.format("filename: %s fileByeSize: %d destination: %s",
```

Comment:
There's a typo. Use MoreObjects.toStringHelper or ask IntelliJ to generate this for you, like in other classes?